### ISSUE TITLE
I have tried on the nuscense dataset. The results seem not well.

### ISSUE THREAD

**Issue by didiwei60** [tsunghan-wu/RandLA-Net-pytorch#28]
![1](https://github.com/tsunghan-wu/RandLA-Net-pytorch/assets/113085596/35074e20-e159-4601-9623-1c1d2cbd1204)
![2](https://github.com/tsunghan-wu/RandLA-Net-pytorch/assets/113085596/f473f89b-c3da-4beb-9920-368f487864af)
I have trained 30 epoches. The best accuracy and iou is only 33.75% and 23.16%, respectively. 
I have used 1800 samples in Nuscense dataset. Each sample contains 34000 points.
The learning rate is set to 0.001. CosineAnnealingLR strategy is token..
I wonder if anything wrong with me. Thank you very much.
